scripts/build/create_commands.py

```diff
@@ -24,6 +24,8 @@ def copy_entry_points(input_data, build_dir):
     """
     package_root = os.path.join(build_dir, 'CodeChecker')
     package_bin = os.path.join(package_root, 'bin')
+    package_bin_cmd = os.path.join(package_root, 'bin', 'cmd')
+    os.makedirs(package_bin_cmd)
```

Comment:
If I run the `make package` target twice without `clean_package` target it will give me the following exception:
```sh
Traceback (most recent call last):
  File "./scripts/build/create_commands.py", line 118, in <module>
    copy_entry_points(args['input'], args['build_dir'])
  File "./scripts/build/create_commands.py", line 28, in copy_entry_points
    os.makedirs(package_bin_cmd)
  File "~/codechecker/venv_dev/lib/python3.6/os.py", line 220, in makedirs
    mkdir(name, mode)
FileExistsError: [Errno 17] File exists: '~/codechecker/build/CodeChecker/bin/cmd'
```